analyses/tcga-capture-kit-investigation/scripts/boxplot.R

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env Rscript
+library(ggplot2)
+
+args = commandArgs(trailingOnly=TRUE)
+
+## read data and rename df header
+dat <- read.table(args[1], sep = "\t")
+names(dat) <- c('sample','mutation','project','caller','tumor_type')
+
+## generate combined boxplot as well as plots for each caller
+boxplot_all <- ggplot(
```

Comment:
This boxplot is probably sufficient; no need to have the 3 separate files that I can see. (If this were a finalized analysis, I would also suggest making this repeated code into a function, rather than retyping it with minimal changes)

Reply:
the boxplot-all has the same x-axis/tumor-type for all 3 callers there as it was only sorted by the overall median, not for each caller's separately. hence i created the other three. but agin, agreed, i think the idea is here to have a overall look of the distribution so we can move on the next step of re-run the those samples.

I commented the other three plots, and updated related readme too

Reply:
two commit for this, https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/597/commits/52e6bbe45dcf0414b0e48653930662ed434bce34 and https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/597/commits/b374755aa67cfc357c5c50c8c1e0a55261922459 Please Review